content/plugins/gcp.md

```diff
@@ -637,55 +663,55 @@ A GCP Instance (i.e. a VM).
 **Properties:**
 
 
-  * `scopes`
-    Optional scopes. If not will set by default:  'https://www.googleapis.com/auth/devstorage.read_write', 'https://www.googleapis.com/auth/logging.write'
+  * `scopes` - 
+    Optional scopes. If not, the type is derived from:  'https://www.googleapis.com/auth/devstorage.read_write', 'https://www.googleapis.com/auth/logging.write'
```

Comment:
"default" is more correct than "derived from" here. It's just a list of strings not a type